bciers/apps/registration/app/data/jsonSchema/operationRegistration/operationInformation.tsx

```diff
@@ -0,0 +1,167 @@
+  purposeNote,
+  purposePreface,
+} from "./operationInformationText";
+
+export const operationInformationSchema: RJSFSchema = {
+  title: "Operation Information",
+  type: "object",
+  required: ["registration_purpose", "operation"],
+  properties: {
+    purpose_preface: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      readOnly: true,
+    },
+    registration_purpose: {
+      type: "string",
+      title: "The purpose of this registration is to register as a:",
+      anyOf: [],
+    },
+    purpose_note: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      readOnly: true,
+    },
+    operation_preface: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      readOnly: true,
+    },
+    operation: {
+      type: "string",
+      title: "Select your operation:",
+      anyOf: [],
+    },
+  },
+  dependencies: {
+    registration_purpose: {
+      allOf: [
+        {
+          if: {
+            properties: {
+              registration_purpose: {
+                const: "OBPS Regulated Operation",
+              },
+            },
+          },
+          then: {
+            required: ["regulated_products"],
+            properties: {
+              regulated_products: {
+                type: "array",
+                items: {
+                  type: "number",
+                },
+                title: "Regulated Product Name(s):",
+              },
+            },
+          },
+        },
+        {
+          if: {
+            properties: {
+              registration_purpose: {
+                const: "Reporting Operation",
+              },
+            },
+          },
+          then: {
+            required: ["regulated_products"],
+            properties: {
+              regulated_products: {
+                type: "array",
+                items: {
+                  type: "number",
+                },
+                title: "Regulated Product Name(s):",
+              },
+            },
+          },
+        },
+        {
+          if: {
+            properties: {
+              registration_purpose: {
+                const: "New Entrant Operation",
+              },
+            },
+          },
+          then: {
+            required: ["regulated_products"],
+            properties: {
+              regulated_products: {
+                type: "array",
+                items: {
+                  type: "number",
+                },
+                title: "Regulated Product Name(s):",
+              },
+            },
+          },
+        },
+        {
+          if: {
+            properties: {
+              registration_purpose: {
+                const: "Opted-in Operation",
+              },
+            },
+          },
+          then: {
+            required: ["regulated_products"],
```

Comment:
@BCerki Did you notice the following error? 
Possibly related to `required isn't working` ?

```
retrieveSchema.js:279 could not merge subschemas in allOf:
 Error: No resolver found for key format. You can provide a resolver for this keyword in the options, or provide a default resolver.
```

https://rjsf-team.github.io/react-jsonschema-form/docs/advanced-customization/internals/#json-schema-supporting-status

Reply:
Where are you seeing it? I haven't got it in my terminals and safejsonparse didn't pick it up

Reply:
@BCerki 
After `Save and Continue` click event in browser console:

![image](https://github.com/user-attachments/assets/070d2333-5b04-4160-ade4-5575aae17a37)

Reply:
Thanks! I still don't see it in my console but the screenshot was enough to figure out it's an open issue with rjsf: https://github.com/rjsf-team/react-jsonschema-form/issues/2752. So I'll switch to `oneOf` instead of if/then and that ~should solve it

Reply:
`oneOf` helped, but it turns out we have an existing bug in the MultiSelectWidget that prevents it from being required: https://github.com/bcgov/cas-registration/issues/1602. I'll add a note to that ticket to comment back in code from this PR